contracts/data/Keys.sol

```diff
@@ -1998,4 +2000,10 @@ library Keys {
             token
         ));
     }
+
+    // @dev key for the buyback max price age
+    // @return key for buyback max price age
+    function buybackMaxPriceAgeKey() internal pure returns (bytes32) {
```

Comment:
if there is no params required for the key, then usually we just refer to it directly, so a function is not needed, e.g.: https://github.com/gmx-io/gmx-synthetics/blob/main/contracts/oracle/Oracle.sol#L244

Reply:
Ok, this has been removed in the latest commit and the base key is now referenced directly in FeeHandler.sol